tests/general/LineContinuationBearTest.py

```diff
@@ -12,6 +12,12 @@
 good_file = """
 a = some_function(
     '1' + '2')
+
+def fun():
```

Comment:
And this as well? You should only add new test cases/modify existing test cases.

Reply:
This is the case where such a doctest exists. Therefore, the bear must ignore backslashes in the line inside doctest containing `from`. Hence, I have added this function in `good_file`.